app/src/modules/content/components/comparison-header.vue

```diff
@@ -0,0 +1,302 @@
+<script setup lang="ts">
+import { userName } from '@/utils/user-name';
+import { localizedFormat } from '@/utils/localized-format';
+import { User } from '@directus/types';
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+interface Props {
+	title: string;
+	dateUpdated: Date | null;
+	userUpdated: User | null;
+	userLoading: boolean;
+	showDeltaDropdown?: boolean;
+	comparisonData?: any;
+	showLatestChip?: boolean;
+	loading?: boolean;
+}
+
+const { t } = useI18n();
+
+const props = defineProps<Props>();
+
+const emit = defineEmits<{
+	(e: 'delta-change', value: number): void;
+}>();
+
+const userUpdatedName = computed(() => {
+	if (!props.userUpdated) return null;
+	return userName(props.userUpdated);
+});
+
+const deltaOptions = computed(() => {
+	if (!props.showDeltaDropdown || !props.comparisonData?.selectableDeltas) return [];
+
+	const deltas = props.comparisonData.selectableDeltas;
+	return deltas.map((delta: any) => {
+		if (delta.activity?.timestamp) {
+			const date = localizedFormat(new Date(delta.activity.timestamp), String(t('date-fns_date_short')));
+			const time = localizedFormat(new Date(delta.activity.timestamp), String(t('date-fns_time')));
+			const formattedDate = `${date} (${time})`;
+
+			let user = t('private_user');
+
+			if (typeof delta.activity.user === 'object') {
+				user = userName(delta.activity.user);
+			}
+
+			return {
+				text: `${formattedDate} - ${user}`,
+				value: delta.id,
+			};
+		}
+
+		return {
+			text: delta.name || delta.key || `Item ${delta.id}`,
+			value: delta.id,
+		};
+	});
+});
+
+const selectedDeltaId = computed(() => {
+	if (!props.showDeltaDropdown || !props.comparisonData) return null;
+
+	return props.comparisonData.initialSelectedDeltaId || null;
+});
+
+const selectedOption = computed(() => {
+	if (!selectedDeltaId.value) return null;
+	return deltaOptions.value.find((option: any) => option.value === selectedDeltaId.value);
+});
+</script>
+
+<template>
+	<div class="comparison-header">
+		<div class="header-content">
+			<div class="title-container">
+				<template v-if="loading">
+					<v-skeleton-loader type="text" class="title-skeleton" />
+				</template>
+				<template v-else>
+					<h3>{{ title }}</h3>
+					<v-chip v-if="showLatestChip" small class="latest-chip">{{ t('latest_version') }}</v-chip>
+				</template>
+			</div>
+		</div>
+		<div class="header-meta">
+			<template v-if="loading">
+				<div class="meta-skeleton">
+					<v-skeleton-loader type="text" class="date-skeleton" />
+				</div>
+			</template>
+			<template v-else>
+				<div v-if="showDeltaDropdown" class="delta-dropdown">
+					<v-menu show-arrow>
```

Comment:
Let’s let the menu appear attached to the button.

```suggestion
					<v-menu attached>
```